### PR TITLE
emily_landi_simple-test-and-modular-patterns

### PR DESCRIPTION
**Pull request by emily2124**

### CODE REVIEW

File: emily_landi/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+**/node_modules
```
Comment (RyuGuardian) [inline emily_landi/.gitignore]: I think the .gitignore needs to be in the directory above this (should be in the root directory--the folder that's made when you run "git clone".) The node_modules folder was still added. If you remove it, I'll merge your pull request.

Comment (RyuGuardian) [inline emily_landi/.gitignore:1]: Although you added node_modules to .gitignore, the folder was still committed (did you add the line to .gitignore after committing?). If you remove node_modules from your submission, I'll merge your pull request. (The .DS_Store file should also be in your .gitignore.)

File: emily_landi/lib/greet.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+//ORIGINAL
+// module.exports = function(name){
+//   return 'hello ' + name;
+// };
+
+//BONUS
+var input = process.argv[2];
+console.log("input: " + input);
```
Comment (RyuGuardian) [inline emily_landi/lib/greet.js:9]: console.logs that aren't part of functionality should be taken out before submitting code.

File: emily_landi/lib/greet.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+//ORIGINAL
+// module.exports = function(name){
```
Comment (RyuGuardian) [inline emily_landi/lib/greet.js]: Keeping commented out code is understandable for what your working on (in assignments or branches) but when submitting for pull requests (or committing to master), it should be left out.

File: emily_landi/lib/greet.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+//   return 'hello ' + name;
+// };
+
```
Comment (RyuGuardian) [inline emily_landi/lib/greet.js:6]: Keeping commented out code is understandable for what your working on (in assignments or branches) but when submitting for pull requests (or merging with master), it should be left out.

File: emily_landi/.gitignore (hunk 1 above)
Comment (emily2124) [inline emily_landi/.gitignore:1]: node_modules has been removed.

Comment (RyuGuardian) [inline emily_landi/.gitignore:1]: Thanks!

**Pull request merged**